src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -369,22 +369,23 @@
       Overwrite="true"
       Encoding="Ascii" />
 
-      <!--======================================================
+    <!--======================================================
                       Section for netfx test runs
           ====================================================== -->
     <ItemGroup Condition="'$(BuildingNETFxVertical)' == 'true'" >
       <TestCommandLines Include="set DEVPATH=%RUNTIME_PATH%" />
     </ItemGroup>
 
-        <!--======================================================
-                      Section for uapaot ilc test runs
+    <!--======================================================
+                      Section for aot ilc test runs
             ====================================================== -->
     <PropertyGroup Condition="'$(UseDotNetNativeToolchain)' == 'true'">
       <ILCBuildType Condition="'$(ILCBuildType)' == ''">ret</ILCBuildType>
       <_UseSharedAssemblies Condition="'$(EnableMultiFileILCTests)' == 'true'">-useSharedAssemblies</_UseSharedAssemblies>
       <_TestILCFolder>%RUNTIME_PATH%\TestILC</_TestILCFolder>
       <_Runtime_Path>%RUNTIME_PATH%\ILCInputFolder\</_Runtime_Path>
-      <!-- Currently (and probably forever) we can't build UAPAOT on ARM,
+      <_ILCWin32 Condition="'$(_bc_TargetGroup)'!='uapaot'">-win32</_ILCWin32>
```

Comment:
Instead of using _bc_TargetGroup here could you use `BuildingUAPAOTVertical != true`?

Reply:
Done